templates/outerDescribe.handlebars

```diff
@@ -1,5 +1,9 @@
 'use strict';
 var chai = require('chai');
+/*eslint-disable*/
+var ZSchema = require('z-schema');
```

Comment:
I believe `var ZSchema = require('z-schema');` can be outside of eslint-disabled area.